base_classes/NXdata.nxdl.xml

```diff
@@ -446,21 +446,49 @@
 	</field>
 
 	<!-- Data vs. plot coordinates -->
-	<field name="scaling_factor" type="NX_FLOAT">
+	<field name="FIELDNAME_scaling_factor" type="NX_FLOAT">
```

Comment:
Should this be DATA to align with changes in #1396

Reply:
I brought in the language from #1396.  FIELDNAME to me makes the most sense as one could apply scaling factor and offset to other parts of the file.  Or am I reading too broadly here?